src/fitsviewer.py

```diff
@@ -149,8 +149,9 @@ def reset(self):
         self.update_hold()
 
     def update_hold(self):
+        pass
 
-        self.canvas.axes.hold(self.ui.keepPrevious.isChecked())
+        #self.canvas.axes.hold(self.ui.keepPrevious.isChecked())
```

Comment:
Why did you comment this, does it not work any more? Can't it be fixed instead?

Reply:
Yes, it dosen't work any more.  Everything works after being commented.